src/pint/fitter.py

```diff
@@ -348,13 +362,22 @@ def get_summary(self, nodmx=False):
                     t0 = tasc + pb * om / 360.0
                     s += "T0  = {:SP}\n".format(t0)
 
-                    s += pint.utils.ELL1_check(
-                        self.model.A1.quantity,
-                        ecc.nominal_value,
-                        self.resids.rms_weighted(),
-                        self.toas.ntoas,
-                        outstring=True,
-                    )
+                    if is_wideband:
+                        s += pint.utils.ELL1_check(
+                            self.model.A1.quantity,
+                            ecc.nominal_value,
+                            self.resids.toa.rms_weighted(),
```

Comment:
Does the `ELL1` condition really not depend on the DM residuals? It makes sense but I just wanted to check.

Reply:
Well, not in a direct way.  The DM data will influence the TOA residuals (typically the TOA RMS will be larger), but I don't think the baseline ELL1_check needs to be changed.